packages/react-slider/src/utils/getRTLSafeKey.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Finds and swaps a provided key for it's right to left format.
```

Comment:
Seems like this should live in `react-utilities` maybe?

Reply:
I think that it could makes sense to move `getRTLSafeKey` and `clamp` to react-utilities. Ideally moving these out of Slider would be a separate PR and we can keep the scope lower here.

Reply:
Here is the pr for moving `getRTLSafeKey `, `clamp`, and `on` into react-utilities
https://github.com/microsoft/fluentui/pull/19852